gnome-initial-setup/pages/language/Makefile.am

```diff
@@ -19,6 +19,7 @@ BUILT_SOURCES += language-resources.c language-resources.h
 
 libgislanguage_la_SOURCES =				\
 	cc-common-language.c cc-common-language.h 	\
+	cc-util.c cc-util.h 						\
```

Comment:
It's used in previous commits. It's possible that the github web UI confuses the order of commits.